recipes/asimov/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "asimov" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 160df0c977cf5c65794f244957c7f4436d250bbf63fd944654219bf29b9114e3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - locutus=asimov.locutus:cli
+    - olivaw=asimov.olivaw:olivaw
+    - asimov=asimov.olivaw:olivaw
+  skip: true  # [win]
```

I think this can be `noarch: python` even though it will be uninstallable on Windows:

```suggestion
build:
  noarch: python
  number: 0
  script: {{ PYTHON }} -m pip install . -vv
  entry_points:
    - asimov = asimov.olivaw:olivaw
    - locutus = asimov.locutus:cli
    - olivaw = asimov.olivaw:olivaw
```

---

Hmm, this threw a CI error when I tried it.

---

I've switched back to it anyway.

---

Yes, the Windows job will fail, but that's ok as long as the failure is understood.